nginx.conf

```diff
@@ -167,4 +167,57 @@ http {
             return 403;
         }
     }
+
+      map $uri:$status:$request_method $loggable_module {
+            ~(\.tar\.gz)\:200\:GET$ 1;
+            default 0;
+         }
+
+        map $uri $module {
+            ~/modules/(?<captured_module>[^/]*)(.*(\.tar\.gz)) $captured_module;
+        }
+
+        log_format json escape=json
+            '{'
+            '"time_local":"$time_local",'
+            '"module":"$module",'
+            '"upstream_addr":"$upstream_addr",'
+            '"request":"$request",'
+            '"body_bytes_sent":"$body_bytes_sent",'
+            '"request_time":"$request_time",'
+            '"http_user_agent":"$http_user_agent"'
+            '}';
+
+       server {
+            listen 81;
+            server_name default_server;
```

Comment:
Shouldn't this say something else than `default_server`? Or is it OK because it's on a separate port?

Reply:
AFAIK yes, it is ok because of different ports